ActindoWorkFlow/Operation/06_InsertStaticInput.md

```diff
@@ -0,0 +1,63 @@
+# Insert a static input
+
+With the static input function, you can use an input port to pass customer-specific data to a transition, with which you want to process it. Only valid JSON values are accepted, for example text ("text"), numbers ("42") or JSON objects ({"dispatchNote": {"id": 42}}).  
+For example, you want to send offer data to a specific sales channel. Since it is not possible to input a sales channel to a start place and to pass it through the entire workflow, you can input this information to a transition that then passes the sales channel information to an API endpoint.   
```

Comment:
Ich verstehe diesen Satz nicht ganz bzw. bin mir nicht sicher, ob es so stimmt. In der Regel ist es so, wenn ich es richtig verstehe, dass man das Angebot aus einem Produkt erstellt wird. Um ein Angebot zu erstellen braucht man das Produkt (input 1) und eine Verbindung zum jeweiligen Shop (input 2, hier erklärt). Es ist nicht, dass man das will, es ist einfach benötigt. Die Verbindung kann man auch anders in den input eingeben (mit mehreren Transitions, die dann die Verbindung-ID "rausspucken"), aber mit static input ist deutlich einfacher. 

API endpoint = transition, oder? Ich denke, das hier so zu erklären verwirrt eher als hilft.